openprescribing/dmd/management/commands/bulk_import_drug_tariff.py

```diff
@@ -0,0 +1,72 @@
+"""Bulk importer for manually-prepared tariff CSV.
+
+
+This probably won't be used again following initial data load, so
+could be deleted after that.
+
+"""
+import csv
+import logging
+from datetime import datetime
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from dmd.models import TariffPrice
+from dmd.models import DMDVmpp
+from dmd.models import DMDProduct
+from frontend.models import ImportLog
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    args = ''
+    help = 'Imports a CSV of historic tariff prices'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--csv')
+
+    def handle(self, *args, **options):
+        '''
+        '''
+        with open(options['csv'], 'r') as f:
+            with transaction.atomic():
+                month = None
+                counter = 0
+                for row in csv.DictReader(f):
```

Comment:
Unless the file is massive, I'd prefer:

```
with open(path) as f:
    rows = list(csv.DictReader(f)

for row in rows:
    ...
```

This reduces nesting.

Reply:
agreed but ignoring as the entire file becomes redundant after deploy!